src/Tribe/Template/Day.php

```diff
@@ -185,14 +184,36 @@ public function ajax_response() {
 					$args[ Tribe__Events__Main::TAXONOMY ] = $_POST['tribe_event_category'];
 				}
 
-				/** @var WP_Query query */
-				$query = tribe_get_events( $args, true );
+				$event_date = tribe_get_request_var( 'eventDate', '' );
+				if ( empty( $event_date ) ) {
+					$event_date = date( 'Y-m-d', current_time( 'timestamp' ) );
+				}
+
+				$args['posts_per_page'] = - 1; // show ALL day posts
```

Comment:
I think this unary operator should be close to the number to make it more clear that is `-1` instead of `- 1`